apps/ledger-live-desktop/tests/utils/allureUtils.ts

```diff
@@ -14,6 +17,16 @@ export async function captureArtifacts(page: Page, testInfo: TestInfo) {
   const screenshot = await page.screenshot();
   await testInfo.attach("Screenshot", { body: screenshot, contentType: "image/png" });
 
+  try {
```

Comment:
do we actually need try/catch here if it's already present in takeScreenshot(); method?